### PR TITLE
Updated version of Insider control files

### PR DESCRIPTION
**Pull request by DaveTryon**
#### Describe the change
Updated version of Insider control files (still needs more detailed copy)

#### PR checklist

- [ ] Run through of all [test scenarios](https://github.com/Microsoft/accessibility-insights-windows/blob/master/docs/Scenarios.md) completed?
- [ ] Does this address an existing issue? If yes, Issue# - 
- [ ] Includes UI changes?
  - [ ] Run the production version of Accessibility Insights for Windows against a version with changes.
  - [ ] Attach any screenshots / GIF's that are applicable.

> Note: After the PR has been created, certain checks will be kicked off. All of these checks must pass before a merge.

### CODE REVIEW

File: Channels/Insider/release_notes.md [hunk 1]
```diff
@@ -1,9 +1,26 @@
-## April 2019 Insider Release ([v1.1.0822.06](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0822.06))
+## April 2019 Insider Release ([v1.1.0828.01](https://github.com/Microsoft/accessibility-insights-windows/releases/tag/v1.1.0828.01))
+
+***Note: This is a preliminary version that needs more detailed copy***
 
 Welcome to the April 2019 release of Accessibility Insights for Windows. Key highlights include:
```
Comment (ferBonnin) [inline Channels/Insider/release_notes.md]: Highlights should be an H2

File: Channels/Insider/release_notes.md [hunk 2]
```diff
@@ -1,9 +1,26 @@
...
+GitHub users can now file issues directly from inside Accessibility Insights for Windows. Bug filing from Azure DevOps is still supported, and you can freely switch between the two.
+
+### Multiple Release Channels
+
+You can now choose how frequently their client updates:
+- The **Production** channel updates every 2-4 weeks
+- The **Insider** channel updates every 1-2 weeks
+- The **Canary** channel updates multiple times per week
+
+You'll be in the **Production** channel by default, but you can change your channel at any time via the **Settings** page
```
Comment (ferBonnin) [inline Channels/Insider/release_notes.md]: probably would be good to add an image on this and a pointer to the documentation on how to change channel

File: Channels/Insider/release_notes.md [hunk 3]
```diff
@@ -1,9 +1,26 @@
...
 
-### GitHub issue filing
+### Rule Updates
 
-GitHub users can now file issues directly from inside Accessibility Insights for Windows. (Need Copy here)
+We have updated several of our accessibility rules based on community feedback.
```
Comment (ferBonnin) [inline Channels/Insider/release_notes.md]: do we have a list of rules?

File: Channels/Insider/release_notes.md (hunk 2 above)
Comment (DaveTryon) [reply Channels/Insider/release_notes.md]: That's part of the work to move from preliminary to final

File: Channels/Insider/release_notes.md (hunk 3 above)
Comment (DaveTryon) [reply Channels/Insider/release_notes.md]: At this point, only a couple of rules have been modified. Rob and Karan are working on more, but they aren't included in the current release

Comment (neocln) [reply Channels/Insider/release_notes.md]: could you make sure that at least the two known user reported false positive fixes are included when a later build is promoted to Insider?

**Pull request merged**